arrow/buffer/bitmap.mojo

```diff
@@ -24,7 +24,8 @@ struct Bitmap(StringableRaising):
     ```
     """
 
-    var _buffer: Pointer[UInt8]
+    alias _ptr_type = DTypePointer[DType.uint8]
+    var _buffer: Self._ptr_type
```

Comment:
```suggestion
    var _buffer: DTypePointer[DType.uint8]
```

Reply:
Other places in the code use the Self._ptr_type to construct a ptr so that the type is changeable in 1 definition. If you still want it to change I'll change it np.